portfolio/src/main/webapp/style.css

```diff
@@ -1,55 +1,71 @@
-#about-title {
-    font: 32px Arial, Helvetica, sans-serif;
-    font-weight: bold;
-    text-align: center;
-    text-decoration: underline;
-}
-.about {
-    font: 16px/2 Arial, Helvetica, sans-serif;
-    margin: 35px 350px;
-}
-
 body,
 html {
     margin: 0;
     padding: 0;
 }
 
+.jumbotron {
+    align-items: center;
+    background-image: linear-gradient(rgba(0,0,0,0.7),rgba(0,0,0,0.7)),url("/images/jumbo4.jpg");
```

Comment:
Linting: Spaces after commas

Reply:
Got it. Spacing is probably the biggest adjustment for me in regards to the style guide, it's easy to forget until its a habit.